keys.js

```diff
@@ -23,23 +24,23 @@ const keys = {
    * @param {Buffer} seed
    */
   deriveRootMetaFeedKeyFromSeed(seed) {
-    return keys.deriveFeedKeyFromSeed(seed, 'metafeed', 'bendy butt')
+    return keys.deriveFeedKeyFromSeed(seed, 'metafeed', 'bendybutt-v1')
```

Comment:
It makes underlying code easier to handle if the format is `bendybutt-v1` because then it matches exactly what SSB URI and BFE expect, and also I suspect that the space in `bendy butt` could have surprised some developer in the future